src/Sylius/Bundle/CoreBundle/Validator/Constraints/CatalogPromotionRulesValidity.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+final class CatalogPromotionRulesValidity extends Constraint
```

Comment:
Eligibility IMO is much more consistent with the rest.

```suggestion
final class CatalogPromotionRulesEligibility extends Constraint
```

Reply:
But the meaning is not the same 😄 As far as I see constraint names in the `CoreBundle` are not consistent, we have both names with the verb (`HasEnabledEntity`) and describing a target entity state in passive voice (`RegisteredUser`). Following the convention used in Symfony, I believe we should go with the latter one. So it should be something like `ValidCatalogPromotionRule` 🖖